wallet/tests/e2e/specs/e2e-spec.js

```diff
@@ -0,0 +1,252 @@
+/* eslint-disable cypress/no-unnecessary-waiting */
+
+/*
+ * This is the e2e test for nightfall browser
+ * Note: to run it locally, follow below steps
+ *  1. start all the containers from scratch.
+ *  2. start wallet/nightfall_browser app (follow readme.md)
+ *  3. In different terminal in wallet/ dir, run `npm run e2e-test`
+ */
+
+describe('End to End tests', () => {
+  let currentTokenBalance = 0;
+  const depositValue = 4;
+
+  beforeEach(() => {
+    cy.on('window:before:load', win => {
+      cy.spy(win.console, 'log');
+      cy.spy(win.console, 'error');
+      cy.spy(win.console, 'warn');
+    });
+  });
+
+  context('MetaMask', () => {
+    it('getNetwork should return network by default', () => {
+      cy.getNetwork().then(network => {
+        expect(network.networkName).to.be.equal('ganache-nightfall');
```

Comment:
This test should work with non ganache deployment too.
So this shouldn't be hardcoded.

Reply:
actually its just the environment variable check, i set here
 ```
 "e2e-test": "NETWORK_NAME=ganache-nightfall RPC_URL=http://localhost:8546 CHAIN_ID=1337 PRIVATE_KEY=0x4775af73d6dc84a0ae76f8726bda4b9ecf187c377229cb39e1afa7a18236a69e npx synpress run"
 ```
 code is in wallet/package.json